packages/portal-proto/src/features/annotations/registerApp.ts

```diff
@@ -0,0 +1,15 @@
+import { createGdcAppWithOwnStore } from "@gff/core";
+import { AppContext, AppStore, id } from "./appApi";
+import AnnotationsBrowser from "./AnnotationBrowser";
+
+export default createGdcAppWithOwnStore({
+  App: AnnotationsBrowser,
+  id: id,
+  name: "Annotations Browser",
+  version: "v1.0.0",
+  requiredEntityTypes: [],
+  store: AppStore,
+  context: AppContext,
+});
+
+export const AnnotationBrowserAppId: string = id;
```

Comment:
do we need to make any changes here now that it's been un-apped? @amlehman 
I saw some message about:

> portal-proto: Module not found: Can't resolve './AnnotationsBrowser' in '/Users/wendyteo/code/gdc-frontend-framework/packages/portal-proto/src/features/apps'

Reply:
Where are you seeing that message @wteouchicago ? I'm not seeing it in the console or the build.

Reply:
@amlehman 😵 it showed up when I ran `npm run dev` but it's gone now after I switched branches and back. false alarm?